pyramid_oereb/lib/records/view_service.py

```diff
@@ -216,80 +209,89 @@ def get_bbox(geometry):
             geom_bounds[3] + height_to_add,
         ]
 
-    def get_full_wms_url(self, real_estate, format):
+    def get_full_wms_url(self, real_estate, image_format, language):
         """
         Returns the WMS URL to get the image.
 
         Args:
             real_estate (pyramid_oereb.lob.records.real_estate.RealEstateRecord): The Real
                 Estate record.
-            format (string): The format currently used. For 'pdf' format,
-                the used map size will be adapted to the pdf format,
+            image_format (string): The format currently used. For 'pdf' format,
+                the used map size will be adapted to the pdf format.
+            language (string): which language of the reference WMS should be used
 
         Returns:
             str: The url used to query the WMS server.
         """
 
         assert real_estate.limit is not None
 
-        map_size = self.get_map_size(format)
+        map_size = self.get_map_size(image_format)
         bbox = self.get_bbox(real_estate.limit)
-        self.reference_wms = add_url_params(self.reference_wms, {
-            "BBOX": ",".join([str(e) for e in bbox]),
-            "SRS": 'EPSG:{0}'.format(Config.get('srid')),
-            "WIDTH": int(map_size[0]),
-            "HEIGHT": int(map_size[1])
-        })
-        self.calculate_ns()
+
+        if language not in self.reference_wms:
+            msg = f"No WMS reference found for the requested language ({language}), using default language"
```

Comment:
I didn't know about this formatting possibility. Nice, I learned something today 👍